rct/client.py

```diff
@@ -161,3 +166,7 @@ def download_disk(base_url, auth_key, disk_path, rct_id, out_file,
     if ranges:
         get_disk_content(base_url, auth_key, disk_path, out_file, ranges,
                          verify=verify)
+        range_len = sum([d["length"] for d in ranges])
```

Comment:
range_len creates a bit of confusion with range_length
what about just putting it in the next line?

total_transfered += sum([d["length"] for d in ranges])